packages/core/demo/data/line.ts

```diff
@@ -77,6 +77,69 @@ export const lineTimeSeriesOptions = {
 	curve: "curveMonotoneX"
 };
 
+export const lineTimeSeriesDenseData = [
+	{ group: "Dataset 1", date: new Date(2019, 0, 1), value: 10000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 1, 5), value: 12000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 1, 10), value: 14000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 2), value: 25000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 2, 2), value: 26000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 3), value: 10000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 3, 5), value: 10000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 3, 10), value: 12000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 5), value: 45000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 7), value: 49000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 7, 15), value: 45000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 9), value: 50000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 9, 5), value: 52000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 9, 15), value: 55000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 10), value: 50000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 12), value: 65000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 13), value: 80000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 14, 10), value: 85000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 15, 7), value: 90000 },
+	{ group: "Dataset 1", date: new Date(2019, 0, 15, 18), value: 70000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 1), value: 20000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 1, 3), value: 22000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 1, 16), value: 24000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 2), value: 35000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 2, 7), value: 36000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 3), value: 20000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 3, 6), value: 20000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 3, 18), value: 22000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 5), value: 62000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 6), value: 52000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 7), value: 52000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 7, 15), value: 52000 },
+	{ group: "Dataset 2", date: new Date(2019, 0, 7), value: 52000 },
```

Comment:
```suggestion

```
accidental duplication? Do we need this? I think the demo does already show how overlapping datapoints are handled between the Datasets... would it even make sense for a dataset to have duplicate data key/value pairs? We can't control users' data and duplicates don't impact the graphic at all but the the tooltip will show all the duplicate points on hover 🤔